etc/profile-m-z/nicotine.profile

```diff
@@ -58,6 +58,8 @@ private-dev
 private-tmp
 
 dbus-user none
```

Comment:
We need to replace `dbus-user none` with `dbus-user filter` for the filtering below to work.

Reply:
Done